### PR TITLE
Add David to release manager role

### PR DESCRIPTION
**Pull request by nabobalis**
Actually need to fill in this info.

### CODE REVIEW

File: project/index.rst [hunk 1]
```diff
@@ -178,12 +178,16 @@ See :ref:`role_descriptions` for more details on each role.
 
     Conor MacBride is a solar physics PhD student at Queen's University Belfast, researching waves in the lower solar atmosphere. Prior to this, he obtained an MPhys degree in Mathematics and Theoretical Physics from the University of St Andrews.
 
-.. card:: Role Unfilled
+.. card:: David Stansby
+    :img_name: conor.png
```
Comment (nabobalis) [inline project/index.rst:182]:
```suggestion
    :img_name: david_stansby.png
```

**Comment by nabobalis** [comment]
@dstansby You will want to review this, we forgot to add you to the other page

File: project/index.rst [hunk 2]
```diff
@@ -178,11 +178,17 @@ See :ref:`role_descriptions` for more details on each role.
 
     Conor MacBride is a solar physics PhD student at Queen's University Belfast, researching waves in the lower solar atmosphere. Prior to this, he obtained an MPhys degree in Mathematics and Theoretical Physics from the University of St Andrews.
 
-.. card:: Role Unfilled
+.. card:: David Stansby
+    :img_name: david_stansby.png
+    :github: dstansby
     :title: Release Manager
+    :aff_name: University College London
+    :aff_link: https://www.davidstansby.com/
```
Comment (nabobalis) [inline project/index.rst:186]:
```suggestion
    :aff_link: https://www.ucl.ac.uk/advanced-research-computing/
```

I am unsure what is best here.

File: project/index.rst [hunk 3]
```diff
@@ -178,11 +178,17 @@ See :ref:`role_descriptions` for more details on each role.
...
+    :date: 29 November 2021
 
-    If you are interested in filling this role see the description:
-    :ref:`role_release-manager`
+    David Stansby is a Research Software Engineer in UCL's Center for Advanced Research Computing (ARC),
+    writing software for a wide spectrum of research groups across the university.
```
Comment (nabobalis) [inline project/index.rst:190]:
```suggestion
    David Stansby is a Research Software Engineer in UCL's Center for Advanced Research Computing (ARC), writing software for a wide spectrum of research groups across the university.
```

File: project/index.rst [hunk 4]
```diff
@@ -178,11 +178,16 @@ See :ref:`role_descriptions` for more details on each role.
 
     Conor MacBride is a solar physics PhD student at Queen's University Belfast, researching waves in the lower solar atmosphere. Prior to this, he obtained an MPhys degree in Mathematics and Theoretical Physics from the University of St Andrews.
 
-.. card:: Role Unfilled
+.. card:: David Stansby
+    :img_name: david_stansby.png
+    :github: dstansby
     :title: Release Manager
+    :aff_name: University College London
+    :aff_link: https://www.davidstansby.com/
```
Comment (dstansby) [inline project/index.rst:186]:
```suggestion
    :aff_link: https://www.ucl.ac.uk/advanced-research-computing/expertise/research-software-development
```
This is probably best?

**Review by dstansby — APPROVED** [review]

**Pull request merged**